src/pages/Components/Select/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Select'
+order: 6
+---
+
+> A select is a selection tool used in forms, allowing user to choose between multiple predefined options.
+
+<preview path="src/pages/Components/Select/previews/Selects" nude="true"></preview>
+
+## Do's and Don'ts
+
+<hintitem>
+  Always use a select with a meaningful label.
+</hintitem>
+
+<hintitem>
+  Always use a select maximum width in relation to the options line length.
+</hintitem>
+
+<hintitem>
+  It's recommended to use shorts option texts.
+</hintitem>
+
+<hintitem>
+ Always add ‘-- ’ and ‘ --’ in before and after a first option used as a placeholder and Capitalise the first letter of a select option.
```

Comment:
Maybe you can split this do section in two different ones.
For example : 
*  Always use ‘—‘ before and after the first option used as a placeholder 
* Capitalise the first letter of a select option

Reply:
"With spaces between the dashes and the text". I agree with the splitting too